src/Sequencer.sol

```diff
@@ -88,84 +91,79 @@ contract Sequencer {
 
     // --- Network Admin ---
     function addNetwork(bytes32 network) external auth {
-        if (networks[network]) revert NetworkExists(network);
+        if (networks.contains(network)) revert NetworkExists(network);
```

Comment:
`add` return a boolean specifying is the action was successfully processed, so u could simplify the code by doing
```
if (!networks.add(network)) revert NetworkExists(network);
emit AddNetwork(network);
```

Reply:
Same in `removeNetwork`, `addJob` and `removeJob`

Reply:
Good catch. Simplified this.